compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -73,6 +73,9 @@ trait AllScalaSettings extends CommonScalaSettings, PluginSettings, VerboseSetti
   )
 
   val wikiSyntax: Setting[Boolean] = BooleanSetting("-Xwiki-syntax", "Retains the Scala2 behavior of using Wiki Syntax in Scaladoc.")
+
+  val jvmargs  = PrefixSetting("-J<flag>", "-J", "Pass <flag> directly to the runtime system.")
+  val defines  = PrefixSetting("-Dproperty=value", "-D", "Pass -Dproperty=value directly to the runtime system.")
```

Comment:
Without these settings parser will emit warning: `bad option '-D...' was ignored`. In scala2 the only usage is here: https://github.com/scala/scala/blob/8a2cf63ee5bad8c8c054f76464de0e10226516a0/src/compiler/scala/tools/nsc/fsc/CompileClient.scala#L36
Should we handle them as well? They are needed for compiler server setup, though since we were not having them handled at all yet, we probably don't need them now, am I correct?

Reply:
> Should we handle them as well? They are needed for compiler server setup, though since we were not having them handled at all yet, we probably don't need them now, am I correct?

Probably it's safer to duplicate scala2 behavior, to do otherwise will probably complicate migration for some projects.